pkg/storage/unified/apistore/store.go

```diff
@@ -339,6 +339,19 @@ func (s *Storage) Get(ctx context.Context, key string, opts storage.GetOptions,
 	if err != nil {
 		return err
 	}
+
+	mmm, err := utils.MetaAccessor(objPtr)
```

Comment:
```suggestion
	meta, err := utils.MetaAccessor(objPtr)
```
or similar -- though I know I am guilty of whatever "mmm" variable this was copied from 😜